src/components/molecules/Darkmode/Darkmode.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { Icon } from 'atoms'
+
+class Darkmode extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      active: false
+    }
+
+    this.css = `
+    .content-wrapper { filter: invert(100%); background: #fefefe; }
+    .avatar, video, button, .body-template img, .file-editor { filter: invert(100%); }
+    `;
+
+    this.store = typeof localStorage === 'undefined' ? null : localStorage;
```

Comment:
@Joe-Pena I would check out how we are using our `storage` class. This is how we are handling localStorage throughout the app.